src/views/docs/components/sidebar.js

```diff
@@ -0,0 +1,133 @@
+import React from "react";
+import Radium from "radium";
+import RadLink from "../../../components/hyperlink";
+import theme from "../../../radium-theme";
+
+class Sidebar extends React.Component {
+  getStyles() {
+    return {
+      container: {
+        padding: `${theme.gutter / 2}px`,
+        backgroundColor: "#050505",
+        backgroundImage: "url(../static/bg-radium.jpg)",
+        backgroundRepeat: "no-repeat",
+        backgroundSize: "cover",
+        backgroundPosition: "bottom left"
+      },
+      header: {
+        display: "flex",
+        flexWrap: "wrap"
+      },
+      heading: {
+        color: theme.lighterGray,
+        fontFamily: theme.sansSerif,
+        width: "100%",
+        marginTop: "10px",
+        WebkitFontSmoothing: "antialiased"
+      },
+      subHeading: {
+        marginTop: "10px",
+        color: theme.lighterGray,
+        fontFamily: theme.monospace,
+        fontSize: ".8rem"
+      },
+      menu: {
+        padding: "0px",
+        listStyle: "none",
+        fontFamily: theme.monospace
+      },
+      menuLi: {
+        fontSize: "1rem",
+        WebkitFontSmoothing: "antialiased"
+      },
+      footer: {
+        display: "flex",
+        flexDirection: "column",
+        justifyContent: "flex-end",
+        flex: "1",
+        padding: "0",
+        fontSize: "14px",
+        fontFamily: theme.monospace
+      }
+    };
+  }
+
+  getLinkStyles(slug) {
+    const linkColor = this.props.currentDocument === slug ?
+      theme.white
+      : theme.lightGray;
+
+    return {
+      color: linkColor,
+      ":hover": {
+        color: theme.white
+      }
+    };
+  }
+
+  render() {
+    const styles = this.getStyles();
+    return (
+      <aside style={[styles.container, this.props.layoutStyles]}>
+        <header style={styles.header}>
+          <h2 style={styles.heading}>Radium</h2>
+          <span style={styles.subHeading}>Radium is a set of tools to manage inline styles on React elements. It gives you powerful styling capabilities without CSS.</span>
+        </header>
+        <ul role="menu" style={styles.menu}>
+        <li style={styles.menuLi}>
+          <RadLink
+            href="/"
+            destination="internal"
+            style={this.getLinkStyles("/")}
+          >
+            Radium Home
+          </RadLink>
+        </li>
+          <li style={styles.menuLi}>
+            <RadLink
+              href="/docs/getting-started"
```

Comment:
This list could be generated from mapping the object and referring to `href=/docs/${slug}` in `src/views/docs/components/radium-files.js` ...but perhaps this is upsetting the router.